python/cuml/dask/neighbors/nearest_neighbors.py

```diff
@@ -40,12 +40,33 @@ def _func_get_i(f, idx):
 class NearestNeighbors(BaseEstimator):
     """
     Multi-node Multi-GPU NearestNeighbors Model.
+
+    Parameters
+    ----------
+    n_neighbors : int (default=5)
+        Default number of neighbors to query
+    batch_size: int (optional, default 1024)
+        Maximum number of queries processed at once. This parameter can
+        greatly affect the throughput of the algorithm.
+    handle : cuml.Handle
+        Specifies the cuml.handle that holds internal CUDA state for
+        computations in this model. Most importantly, this specifies the CUDA
+        stream that will be used for the model's computations, so users can
+        run different models concurrently in different streams by creating
+        handles in several streams.
+        If it is None, a new one is created.
+    verbose : int or boolean, default=False
+        Sets logging level. It must be one of `cuml.common.logger.level_*`.
+        See :ref:`verbosity-levels` for more info.
+
     """
-    def __init__(self, client=None, streams_per_handle=0, **kwargs):
+    def __init__(self, client=None, streams_per_handle=0,
```

Comment:
I notice `n_neighbors` in the docs but it doesn't looks like it's being explicitly passed through here. Are we able to do the same w/ batch_size?

Reply:
I think we could just eliminate batch_size here, right? It'll go through kwargs?

Reply:
If I well understood, this `batch_size` makes it possible to specify a common default value for all MNMG KNN classes as MNMG KNN Cl&Re classes inherit from it. Otherwise the default can differ as it is specified at 3 different places. It would even be possible that a model would use different `batch_size` when calling `kneighbors` and `predict`.

Reply:
I think what we probably want here is to set a default batch_size in the `NearestNeighborsMG.pyx` base class when it hasn't been set explicitly. This would propagate the value to all the the subclasses and keep it out of the wrapper entirely.